itou/metabase/sql/034_candidats_auto_prescription.sql

```diff
@@ -3,6 +3,26 @@ with auto_p as (
         distinct(c.id),
         cd.état,
         c.age,
+        c.total_critères_niveau_1,
+        c.total_critères_niveau_2,
+        c.critère_n1_bénéficiaire_du_rsa,
+        c.critère_n1_allocataire_ass,
+        c.critère_n1_allocataire_aah,
+        c.critère_n1_detld_plus_de_24_mois,
+        c.critère_n2_niveau_d_étude_3_cap_bep_ou_infra,
+        c.critère_n2_senior_plus_de_50_ans,
+        c.critère_n2_jeune_moins_de_26_ans,
+        c.critère_n2_sortant_de_l_ase,
+        c.critère_n2_deld_12_à_24_mois,
+        c.critère_n2_travailleur_handicapé,
+        c.critère_n2_parent_isolé,
+        c.critère_n2_personne_sans_hébergement_ou_hébergée_ou_ayant_u,
+        c.critère_n2_réfugié_statutaire_bénéficiaire_d_une_protectio,
+        c.critère_n2_résident_zrr,
+        c.critère_n2_résident_qpv,
+        c.critère_n2_sortant_de_détention_ou_personne_placée_sous_main,
+        c.critère_n2_maîtrise_de_la_langue_française,
+        c.critère_n2_mobilité,
```

Comment:
Oulà on est d'accord que c'est super fragile et que ça va casser à chaque fois que le C1 change un critère ? Ce qui arrive assez souvent quand même. J'ai pas l'impression qu'il soit possible d'injecter que les colonnes en `c.critère_*` cf ceci ([lien](https://stackoverflow.com/questions/3941156/how-to-select-columns-that-have-name-beginning-with-same-prefix)).

Reply:
De toute façon avec les tables CAP je vais très bientôt mettre les critères dans leur propre table C2.

Reply:
Oui c'est super fragile et on a déjà eu à le changer plusieurs foi mais on a un TB public qui se base sur ces critères et la on m'a demandé de refaire pareil mais version privée...